internal/connect/connection.go

```diff
@@ -81,11 +82,11 @@ func proxyWithAuth(req *http.Request) (*url.URL, error) {
 
 func callHTTP(verb, path string, body []byte, query map[string]string, auth authType) ([]byte, error) {
 	if httpclient == nil {
-		tr := &http.Transport{
-			TLSClientConfig: &tls.Config{InsecureSkipVerify: CFG.Insecure},
-			Proxy:           proxyWithAuth,
-		}
-		httpclient = &http.Client{Transport: tr}
+		// use defaults from DefaultTransport
+		tr := http.DefaultTransport.(*http.Transport).Clone()
```

Comment:
it's really weird that `DefaultTransport` is type `RoundTripper` and not `Transport`

Reply:
this whole `DefaultTransport` and `DefaultClient` idea is weird :o)